### ISSUE TITLE
[Usable- Bot Framework Web Chat - Card Wizard]: 'send' button is not provided after the type your message edit field

### ISSUE THREAD

**Issue by manojs5** [microsoft/BotFramework-WebChat#5359]
### Is it an issue related to Adaptive Cards?

No.

### User Experience:
Users might not know how to submit their input, causing frustration. They may wonder if the form or message they have written has been saved or sent.

### Note:
User credentials should NOT be included in the bug.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### What is the public URL for the website?

https://compulim.github.io/webchat-loader/

### How to reproduce the issue?

1. Open URL: chro and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label and then select open web chat in new window link at bottom. (select 'white labelled' radio button)
3. Navigate to white label experience chat page and type Card Wizard word in type your message text box.
4. Navigate to the search results and fill details and then navigate to submit results.
5. Verify the issue.

### What do you expect?

'send' button should be provided after the type your message edit field

### What actually happened?

'send' button is not provided after the type your message edit field
### Observation:
after typing any input and pressing enter the results are showing.

### Do you have any screenshots or recordings to repro the issue?

![Image](https://github.com/user-attachments/assets/ea4c4f40-67f6-4323-96ee-6c5a0f163720)

**Comment by manojs5** [comment]
#A11ySev4;#Accessibility;;#Benchmark;#BotFrameworkWebChat-Oct24;#HCL;#A11yUsable;#Win11-Edge(Chromium);
